modules/server/src/main/ts/bedrock/auto/DriverLoader.ts

```diff
@@ -97,8 +116,9 @@ export const loadDriver = (browserName: string, settings: DriverSettings): Drive
   }
 };
 
-export const waitForAlive = (proc: ChildProcess, port: number, timeout = 30000): Promise<void> => {
-  const url = 'http://127.0.0.1:' + port + '/status';
+export const waitForAlive = (proc: ChildProcess | null, port: number, timeout: number, path: string): Promise<void> => {
+  const url = 'http://127.0.0.1:' + port + path;
```

Comment:
It might be better to have path be either empty string or `/wd/hub`, and continue to append `/status` here. Then you won't have to specify `'/wd/hub'` twice in `Driver.ts`.

That seems more scalable to other URLs once we start looking at running multiple tests against a single bedrock?